frontend/src/app/app/sidebar.tsx

```diff
@@ -2,7 +2,17 @@ import Image from "next/image";
 import Link from "next/link";
 
 const Sidebar = () => {
-  return (
+  return (<>
+    <aside className="sidebar">
```

Comment:
why would you add this link to the sidebar if you had zero thoughts of putting the welcome quest page on that route?